docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -720,6 +720,10 @@ connector using a {doc}`WITH </sql/create-table-as>` clause.
 * - `orc_bloom_filter_fpp`
   - The ORC bloom filters false positive probability. Requires ORC format.
     Defaults to `0.05`.
+* - `parquet_bloom_filter_columns`
+  - Comma-separated list of columns to use for Parquet bloom filter. It improves
+    the performance of queries using Equality and IN predicates when reading
+    Parquet files. Requires Parquet format. Defaults to `[]`.
```

Comment:
Is the lack of `parquet_bloom_filter_fpp` deliberate?

Ihttps://github.com/trinodb/trino/blob/c72facac42e92430f86d631d773fd75cddd8fc57/lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriterOptions.java#L47

Reply:
Yes, it's intentional. Only `write.parquet.bloom-filter-max-bytes` is a standardized part of the Iceberg spec.